mobile/src/main/java/ch/epfl/sdp/mobile/ui/tournaments/ContestScreen.kt

```diff
@@ -0,0 +1,96 @@
+package ch.epfl.sdp.mobile.ui.tournaments
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.Add
+import ch.epfl.sdp.mobile.ui.Filter
+import ch.epfl.sdp.mobile.ui.PawniesIcons
+import ch.epfl.sdp.mobile.ui.plus
+
+/**
+ * This screen displays all the tournaments of the app.
+ *
+ * @param state the [ContestScreenState] to manage the contents of this composable.
+ * @param modifier the [Modifier] for the composable.
+ * @param key a function which uniquely identifies the list items.
+ * @param contentPadding the [PaddingValues] for this screen.
+ * @param C the type of the [Contest].
+ */
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun <C : Contest> ContestScreen(
+    state: ContestScreenState<C>,
+    modifier: Modifier = Modifier,
+    key: ((C) -> Any)? = null,
+    contentPadding: PaddingValues = PaddingValues()
+) {
+
+  Scaffold(
+      topBar = { TopAppBar() },
+      modifier = modifier.padding(contentPadding),
```

Comment:
You're applying the padding twice : once in the `Scaffold`, and once in the `LazyColumn`.